priv/docker/prometheus/prometheus.yml

```
@@ -0,0 +1,13 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: "prometheus"
+
+    static_configs:
+      - targets: ["localhost:9090"]
+  - job_name: elixir_exporter
+    scheme: "http"
+    static_configs:
+      - targets: ["localhost:4000"]
```

```suggestion
      - targets: ["backend:4000"]
```

You can use the app name to make it work on docker compose.